### DISCUSSION TITLE
cant type whit autosave enabled

### DISCUSSION THREAD

**Discussion by aveat** [Natizyskunk/vscode-sftp#402] [Q&A] [answered]
Hello i have a problem/question, and is that whenever i try to type and it auto saves remotely, it goes back, like autoerase what i just wrote, is any way to keep the sync only when i save so i dont get this annoying effect or autdeleting what i lastly wrote ? sometimes is immposible to write a new word because it just keeps erasing my words!
Thanks guys!

**Comment by Natizyskunk** [comment]
It's seems that there's something wrong with your setup/configuration. It shouldn't behave like that.

**Reply by aveat** [reply]
well i know is that because if i disable the sftp extension the problem dissapears, so i have tried to uninstall and install again, and what configuration you mean ? because a sftp plugin just use a few data to work, like servername, pass, etc. so not much room to problems here, since it connects and store i dont think there is a problem. Also never touched any setting in VSCODE . i just dont know. thank you for your reply.

**Reply by Natizyskunk** [reply]
@aveat Could you open an issue respecting [this](https://github.com/Natizyskunk/vscode-sftp/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=) template please ?

**Reply by aveat** [reply, marked as answer]
i just did thank you
